SignalMessaging/utils/DebugLogger.m

```diff
@@ -106,7 +106,7 @@ - (void)enableTTYLogging
     // This should be redundant with the logic above.
     [logPathSet addObjectsFromArray:self.fileLogger.logFileManager.unsortedLogFilePaths];
     NSArray<NSString *> *logPaths = logPathSet.allObjects;
-    return [logPaths sortedArrayUsingSelector:@selector(compare:)];
+    return [logPaths sortedArrayUsingSelector:@selector((compare:))];
```

Comment:
Huh!  Why was this necessary?

Reply:
Is this to suppress / work around the "multiple matching methods with varying types" warning?

Reply:
It was - I can't imagine adding those parentheses actually makes it safer though, probably just snuck past the linter.